Applications/KNN/README.md

```diff
@@ -3,24 +3,24 @@
 Here is some toy example which is distinguish simple images.
 The Mobile ssd V2 tensor flow lite model is used for the feature extractor and Nearest Neighbor is used for the classifier. All the training and testing is done on the Galaxy S8.
 
-![image](https://github.com/nnstreamer/nntrainer/blob/master/docs/images/08b09a80-ef29-11e9-8303-475fd75f4b83.png)
+![image](https://github.com/nnstreamer/nntrainer/blob/main/docs/images/08b09a80-ef29-11e9-8303-475fd75f4b83.png?raw=true)
```

Comment:
I wonder if we can do like `/docs/images/...`

Reply:
Maybe @lhs8928 can help this out??

Reply:
yes, that can also be done.
@lhs8928 can update it maybe